earth_enterprise/src/fusion/autoingest/AssetGenDaemonH.pl

```diff
@@ -209,12 +211,14 @@ ()
         : AssetImpl(AssetStorage::MakeStorage(ref_, $actualtypearg, "$subtype", $actualinputarg, meta_)),
           ${name}AssetImpl(config_), AssetImplD() { }
 
+public:
```

Comment:
Minor: Just checking - this had to become public so that make_shared could call it, correct?

Reply:
yes, i looked into alternate ways of doing this and they seemed not quite so simple as making it a friend; it seemed a bit too complex for its own good.